checkpoint.go

```diff
@@ -18,11 +18,15 @@ var checkpointCommand = cli.Command{
 		cli.StringFlag{Name: "image-path", Value: "", Usage: "path for saving criu image files"},
 		cli.StringFlag{Name: "work-path", Value: "", Usage: "path for saving work files and logs"},
 		cli.BoolFlag{Name: "leave-running", Usage: "leave the process running after checkpointing"},
-		cli.BoolFlag{Name: "tcp-established", Usage: "allow open tcp connections"},
-		cli.BoolFlag{Name: "ext-unix-sk", Usage: "allow external unix sockets"},
-		cli.BoolFlag{Name: "shell-job", Usage: "allow shell jobs"},
+		cli.BoolFlag{Name: "no-tcp", Usage: "fail on open tcp connections"},
+		cli.BoolFlag{Name: "no-ext-unix-sk", Usage: "fail on external unix sockets"},
+		cli.BoolFlag{Name: "no-shell-job", Usage: "fail on shell jobs"},
+		cli.BoolFlag{Name: "no-file-locks", Usage: "fail on file locks, for safety"},
```

Comment:
I actually am fine with reverting the change in runc, but then we should have this discussion in my docker branch, where I made the same change (actually a more aggressive change) and think its more important.